lib/manager/terraform/lockfile/index.spec.ts

```diff
@@ -0,0 +1,340 @@
+import { readFileSync } from 'fs';
+import _fs from 'fs-extra';
+import { join } from 'upath';
+import * as httpMock from '../../../../test/http-mock';
+import { getName } from '../../../../test/util';
+import { getPkgReleases } from '../../../datasource';
+import { defaultRegistryUrls } from '../../../datasource/terraform-provider';
+import type { UpdateArtifactsConfig } from '../../types';
+import hash from './hash';
+import { updateArtifacts } from './index';
+
+// auto-mock fs
+jest.mock('fs-extra');
```

Comment:
I like to see the usage of our `util/fs` wrapper, so mocking should be much easier, as we can't mock `fs` .